quote-generator/src/app/extensions/components/QuoteName.jsx

```diff
@@ -0,0 +1,28 @@
+import { Button, ButtonRow, Input } from '@hubspot/ui-extensions';
+import React, { useState } from 'react';
+
+export const QuoteName = ({ onNext, defaultName }) => {
+  const [quoteName, setQuoteName] = useState(defaultName);
+
+  return (
+    <>
+      <Input
+        label="Quote title"
+        name="quoteName"
+        value={quoteName}
+        required={true}
+        onChange={setQuoteName}
+      />
+      <ButtonRow>
```

Comment:
Do we need this button row for a single button?

Reply:
This is mostly styles adjustment, so a single `Button` inside `Flex` looks small and doesn't fill full width: 

![image](https://github.com/HubSpot/ui-extensions-examples/assets/57258334/3c005c5a-86c5-4c9c-a050-c9149546b96e)

but not

![image](https://github.com/HubSpot/ui-extensions-examples/assets/57258334/0e1ab84d-68f7-40b6-bcd0-143845f4e762)

Reply:
Can you wrap it `Box` component instead? `ButtonRow` comes with some javascript that you don't want to run if you don't need it